### ISSUE TITLE
Use aoestats.net instead of aoe2.net

### ISSUE THREAD

**Issue by procobain** [LucasJappert/overlay-aoe-de#3]
Hello, with the abandonment of aoe2.net, would it be possible to use aoe2stats.net to get the information that your layout needs?

Hug.

**Comment by LucasJappert** [comment]
@procobain Hello dude! Yes of course, i will check aoe2stats.net and if it 's posible to implement, i'll make the fix. Thank you!

**Comment by kmanan** [comment]
Hi, curious if you were able to fix this?

**Comment by LucasJappert** [comment]
@kmanan Hey bro, i really want to fix it but i'm full of work today. I promise you i'll fix it as soon as i can. Thank you!

**Comment by kmanan** [comment]
Ah! No worries! Completely understand. I wanted to make sure I wasn’t doing something wrong

**Comment by procobain** [comment]
@LucasJappert aoe2.net is back, but they must have changed something, their overlay is not working...

Hug...

**Comment by LucasJappert** [comment]
Yes @procobain ! I saw it today. I will check the update tomorrow and try to fix my overlay. Thank you!

**Comment by qtds35k** [comment]
Hi @LucasJappert , thanks for your amazing work on this project!
I wasn't lucky enough to find it before aoe2.net went down but now that it's back I'm so excited to try this out.

I briefly looked into those js files but couldn't figure out why the client URL breaks.
The URLs in *.js work fine as I don't think aoe2.net changed their APIs.
I noticed, however, they changed the endpoint of the player profile on browser. E.g., 
Before: `https://aoe2.net/#profile-2510223`
After: `https://aoe2.net/#aoe2de-profile-2510223`

Not sure if it matters tho.
Anyways please let me know if I can help on anything.
Maybe just throw out some pointers if you ain't got capacity to dive in at the moment and I'll try pick'em up.

Cheers,

**Comment by procobain** [comment]
@LucasJappert 

This other overlay (in the form of a windows application) made the necessary update for the new ao2.net, I think this makes it easier for you to modify yours, take a look...

https://github.com/CraftySalamander/RTS_Overlay/commit/6439fa63c89b4904ab29bde603c9067b13d1b32b

**Comment by GeigerBC** [comment]
I'm brand new to GitHub, but was this AOE2DE overlay issue ever resolved?  It looks like aoe2.net is up, but I don't know the history of aoe2.net.  I thought it was a cool overlay the last time I used it.

**Comment by procobain** [comment]
@LucasJappert 
I imagine you may be out of time, but as soon as you have one, try to arrange the layout for us, yours is the best.

I had put some changes that another layout made with changes from aoe2.net, maybe you can get ahead a little.

https://github.com/CraftySalamander/RTS_Overlay/commit/6439fa63c89b4904ab29bde603c9067b13d1b32b

Hug.
